openapi-cli/src/main/java/io/ballerina/openapi/cmd/CodeGenerator.java

```diff
@@ -116,8 +117,8 @@ public class CodeGenerator {
      * <p>Method can be user for generating Ballerina mock services and clients</p>
      */
 
-    public void generateBothFiles(GeneratorConstants.GenType type, String definitionPath, String serviceName,
-                                  String outPath, Filter filter, boolean nullable, boolean isResource)
+    public void generateClientAndServiceFiles(String definitionPath, String serviceName,
```

Comment:
How about using `generateClientAndService()` just to be consistent with the other APIs?